### PR TITLE
Jenkins pipeline for PerfScale distributed tracing tests

### PR DESCRIPTION
**Pull request by skordas**

### CODE REVIEW

**Comment by skordas** [comment]
@paigerube14 PTAL

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
-# ingress-perf
+# Distributed Tracing
 
+Jenkins pipeline job which will use [Flexy-install](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Flexy-install/) cluster, install Dirstibuted Tracing operators, run workload and upload results data to ElasticSearch
+
+
```
Comment (paigerube14) [inline README.md:5]: Can you add some information and links to what this script is doing and any background on distributed tracing

**Comment by paigerube14** [comment]
Is this job fully working as expected? I see the below error in the most recent passing run. Is that expected? 

`12-11 15:56:09.119  ./scripts/verify-channels.sh: line 11: podman: command not found`

**Comment by skordas** [comment]
> Is this job fully working as expected? I see the below error in the most recent passing run. Is that expected?
> 
> `12-11 15:56:09.119 ./scripts/verify-channels.sh: line 11: podman: command not found`

Yes - it's expected.
This is some sample script from https://github.com/openshift/distributed-tracing-qe repo (not test script designed to run in jenkins pipeline)
It means jenkins was able to get script and run it.

**Comment by paigerube14** [comment]
@skordas will any current test fully run so you could see the output?

**Comment by paigerube14** [comment]
Things look good here to me, is there anything else you want to add before merging?

**Comment by skordas** [comment]
@paigerube14 Right now I don't have any test scripts which can be used by this pipeline.
Now would be great to have working pipeline, to start running tests and then make updates where needed.

**Pull request merged**